gt/train/ONB_aze_18950706_8/ONB_aze_18950706_8.jpg_line_1546496895959_11.gt.txt

```diff
@@ -1 +1 @@
-im Jn- und Auslanda
\ No newline at end of file
+im Jn- und Auslande
```

Comment:
```suggestion
im In- und Auslande
```

Reply:
Changed them and other typesetting errors,